src/Model/Table/RequestsTable.php

```diff
@@ -131,7 +141,26 @@ public function gc(): void
 
             $conn = $this->getConnection();
             if ($conn->getDriver() instanceof Sqlite) {
-                $conn->execute('VACUUM;');
+                $conn->execute('
+                    PRAGMA auto_vacuum = FULL;
+                    PRAGMA journal_mode = OFF;
+                    PRAGMA synchronous = OFF;
+                    PRAGMA foreign_keys = OFF;
+                    PRAGMA temp_store = MEMORY;
+                    PRAGMA automatic_index = OFF;
+                ');
+
+                if (!$this->shouldGcVacuum()) {
```

Comment:
If we've already passed the rand to do a GC why do we need another rand to do vacuum?

Reply:
So rows are cleared more frequently, but the slow vacuum should be ran only 1% of requests (I made the check to own functions for static analysis tools and if we want to tweak the chance)